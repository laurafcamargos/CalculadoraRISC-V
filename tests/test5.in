abc
-123
-
xyz
5
f
