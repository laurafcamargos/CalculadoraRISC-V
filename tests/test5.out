Digite um número: Erro: valor não é um número inteiro válido
Digite um número: Digite a operação (+, -, *, /, u, f): Digite um número: Erro: valor não é um número inteiro válido
Digite um número: Resultado: -128
Digite a operação (+, -, *, /, u, f): 
